src/SourceMapToolkit.CallstackDeminifier/StringExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace SourcemapToolkit.CallstackDeminifier
+{
+	internal static class StringExtensions
+	{
+		/// <summary>
+		/// String.Split() allocates an O(input.Length) int array, and
+		/// is surprisingly expensive.  For most cases this implementation
+		/// is faster and does fewer allocations.
+		/// There's a copy of this function in StandardFlightManager.cs because
```

Comment:
Is StandardFlightManager.cs some internal source code? Maybe we should remove that reference from here?

Reply:
Good call @christiango.  Should be fixed in the latest iteration.